modules/storages/app/models/storages/storage_file.rb

```diff
@@ -30,14 +32,14 @@ module Storages
   StorageFile = Data.define(
     :id,
     :name,
-    :size,
+    :size, # Integer >= 0
     :mime_type,
-    :created_at,
-    :last_modified_at,
+    :created_at, # Time? DateTime (deprecated)?
+    :last_modified_at, # Time? DateTime (deprecated)?
```

Comment:
❓ What exactly is deprecated?

Reply:
DateTime is deprecated in favor of Time.

Reply:
Why not starting using time directly?